src/coldfront_plugin_api/scim_v2/groups.py

```diff
@@ -86,6 +89,11 @@ def patch(self, request, pk, format=None):
                     signals.allocation_activate_user.send(
                         sender=self.__class__, allocation_user_pk=au.pk,
                     )
+
+                    self._set_user_on_project(
+                        project, user, "Active", "User", False
```

Comment:
Does `enable_notifications` default to false when adding someone from the web dashboard? Where is this default coming from?

Reply:
@knikolla False was just the value I decided to put it. I should have asked you what you'd like it to be. 
Reviewing the Coldfront code, they've set the default `enable_notifications` value to `True`. Should I just leave it like that?

Reply:
Do set this to True. I looked into the production ColdFront and it seems that currently all users on projects have this set to true.